exercises/concept/rpn-calculator/.docs/introduction.md

```diff
@@ -0,0 +1,6 @@
+# Introduction
```

Comment:
Probably worth adding a sentence to this section about what a stack is: like a stack of paper, the last items in are the first ones out.

Reply:
FWIW, the [book](https://doc.rust-lang.org/book/ch04-01-what-is-ownership.html#the-stack-and-the-heap) uses the metaphor of plates to discuss the stack. Perhaps we do that here too for consistency?

Reply:
Should be resolved by 2fae730 & ed5ed0d